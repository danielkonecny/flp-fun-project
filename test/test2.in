A,B,C
a,b
A
A->BaB
A->aB
B->b
