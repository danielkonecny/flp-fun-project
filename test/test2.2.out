A,B,C,<aB>,a'
a,b
A
A->B<aB>
<aB>->a'B
A->a'B
a'->a
B->b
